lib/project_types/script/layers/application/create_script.rb

```diff
@@ -48,7 +43,7 @@ def setup_project(ctx:, language:, script_name:, extension_point:, description:,
               identifiers.merge!(config_ui_file: DEFAULT_CONFIG_UI_FILENAME)
               Infrastructure::ConfigUiRepository
                 .new(ctx: ctx)
-                .create_config_ui(DEFAULT_CONFIG_UI_FILENAME, YAML.dump(DEFAULT_CONFIG))
+                .create_config_ui(DEFAULT_CONFIG_UI_FILENAME, default_config_ui_content(script_name))
```

Comment:
Just curious: Is the `script_name` at this point the `snake_case` version?

Reply:
Yeah right now `script_name` is cleansed to be alphanumeric and snake_cased